Fixed #30437 -- Clarified that urlpatterns can be a sequence. (#11330)
django/django: docs/topics/http/urls.txt

   :setting:`ROOT_URLCONF` setting.

#. Django loads that Python module and looks for the variable

>    ``urlpatterns``. This should be an iterable (e.g. a list or tuple) of

Reviewer: Rather than describing an iterable, WDYT about linking the Python docs:

```suggestion
   ``urlpatterns``. This should be an :term:`iterable` of
```

It seems to me we shouldn't need to re-explain what the Python docs already say.

Author: ok. to change.

Author: changed.
